README.md

```diff
@@ -86,60 +86,60 @@ $ bash build.sh
+| image list       | import image           | <kbd>i</kbd>                                                   |
+| image list       | load image             | <kbd>Ctrl</kbd> + <kbd>l</kbd>                                 |
+| image list       | remove dangling images | <kbd>Ctrl</kbd> + <kbd>d</kbd>                                 |
+| image list       | refresh image list     | <kbd>Ctrl</kbd> + <kbd>r</kbd>                                 |
+| image list       | filter image           | <kbd>f</kbd>                                                   |
+| container list   | inspect container      | <kbd>Enter</kbd> / <kbd>o</kbd>                                |
+| container list   | remove container       | <kbd>d</kbd>                                                   |
+| container list   | start container        | <kbd>u</kbd>                                                   |
+| container list   | stop container         | <kbd>s</kbd>                                                   |
+| container list   | export container       | <kbd>e</kbd>                                                   |
+| container list   | commit container       | <kbd>c</kbd>                                                   |
+| container list   | rename container       | <kbd>r</kbd>                                                   |
+| container list   | refresh container list | <kbd>Ctrl</kbd> + <kbd>r</kbd>                                 |
+| container list   | filter image           | <kbd>f</kbd>                                                   |
+| container list   | exec container cmd     | <kbd>Ctrl</kbd> + <kbd>c</kbd>                                 |
+| volume list      | create volume          | <kbd>c</kbd>                                                   |
+| volume list      | remove volume          | <kbd>d</kbd>                                                   |
+| volume list      | prune volume           | <kbd>p</kbd>                                                   |
+| volume list      | inspect volume         | <kbd>Enter</kbd> / <kbd>o</kbd>                                |
+| volume list      | refresh volume list    | <kbd>Ctrl</kbd> + <kbd>r</kbd>                                 |
+| volume list      | filter image           | <kbd>f</kbd>                                                   |
+| network list     | inspect network        | <kbd>Enter</kbd> / <kbd>o</kbd>                                |
+| network list     | remove network         | <kbd>d</kbd>                                                   |
+| pull image       | pull image             | <kbd>Enter</kbd>                                               |
+| pull image       | close panel            | <kbd>Esc</kbd>                                                 |
+| create container | next input box         | <kbd>↓</kbd> / <kbd>Tab</kbd> / <kbd>Ctrl</kbd> + <kbd>n</kbd> |
```

Comment:
create container panel cannot work in this PR.
If you want use the key bind, please add key bind on create container panel.

https://github.com/skanehira/docui/blob/master/panel/form.go#L42